esque/controller/topic_controller.py

```diff
@@ -56,7 +56,7 @@ def list_topics(self, *, search_string: str = None, sort: bool = True, hide_inte
         if sort:
             topic_names = sorted(topic_names)
 
-        topics = list(map(self.get_cluster_topic, topic_names))
+        topics = list(map(lambda topic_name: self.get_local_topic(topic_name, False), topic_names))
```

Comment:
We misunderstood each other I think. I would have introduce a flag here. So the `list_topic` can be used to either collect all topic_names or all topic objects. and just use `get_cluster_topic` or the `get_local_topic`based on the use case. Than you would just call the method with the flag set in the get topics command. :)